tsdb/index/index_test.go

```diff
@@ -505,3 +506,48 @@ func TestNewFileReaderErrorNoOpenFiles(t *testing.T) {
 	// dir.Close will fail on Win if idxName fd is not closed on error path.
 	dir.Close()
 }
+
+func TestSymbols(t *testing.T) {
+	buf := encoding.Encbuf{}
+
+	buf.PutUvarintStr("something")
+
+	symbolsStart := buf.Len()
+	buf.PutBE32int(204)
```

Comment:
This could all do with comments so you know what field is what.

Reply:
:+1: